ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -60,22 +81,53 @@ def obstacle_cb(self, msg):
         # We will implement it later
         pass
 
-    def get_waypoint_velocity(self, waypoint):
+    def find_next_waypoint(self):
```

Comment:
Nice!
I believe we have a very similar function in `tl_detector.py`, maybe we should consider moving both to some `utils.py` function. Doesn't need to be done in this patch though